language/tools/move-package/src/source_package/parsed_manifest.rs

```diff
@@ -38,7 +39,21 @@ pub struct Dependency {
     pub local: PathBuf,
     pub subst: Option<Substitution>,
     pub version: Option<Version>,
-    pub digest: Option<Vec<u8>>,
+    pub digest: Option<PackageDigest>,
+    pub git_info: Option<GitInfo>,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct GitInfo {
+    /// The git clone url to download from
+    pub git_url: Symbol,
+    /// The git reference -- a commit SHA or a branch name, e.g., 'main'
+    pub git_ref: Symbol,
```

Comment:
Do we need to intern these?

Reply:
We don't need to, but we are currently interning everything else, so for consistency it seemed like interning these as well would be a good idea.